ggmolvis/__init__.py

```diff
@@ -57,6 +57,11 @@
 # update every artist when the frame changes
 @persistent
 def update_frame(scene):
+    # This is necessary to e.g. get the correct object
+    # dimensions after frame change during rendering
+    # see: https://blender.stackexchange.com/questions/61635/object-dimensions-changing-but-not-changing
+    bpy.context.view_layer.update() 
```

Comment:
Changing the update handler to user `frame_change_pre` instead of `frame_change_post` might also fix this issue. They don't mean pre and post changing of the frame, but pre and post updating the rest of the scene after changing the frame: https://docs.blender.org/api/current/bpy.app.handlers.html#bpy.app.handlers.frame_change_pre

I've swapped MN to use frame_change_pre as it's better to do any mesh data updates before the rest of blender attempts to compute any updates (like object dimensions)

Reply:
Changing the handler doesn't help in this case, i.e. calculating the dimension of an object in Blender (line: `size_obj_xyz = np.array(self.object.dimensions)`) which somehow needs a manual view layer update.

Changing it to `frame_change_pre` anyway to be consistent.